compose/backend.yml

```diff
@@ -141,6 +141,7 @@ services:
     environment:
       DB_USERNAME:
       DB_PASSWORD:
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
This one is interesting, it started to fail on some Mac envs. I do not need this flag (Ubuntu). Let's keep it as most of us need it now.